Dockerfile

```diff
@@ -9,6 +9,11 @@ ENV cachet_ver ${cachet_ver:-master}
 
 ENV COMPOSER_VERSION 1.4.1
 
+ADD https://php.codecasts.rocks/php-alpine.rsa.pub /etc/apk/keys/php-alpine.rsa.pub
```

Comment:
Is there any information on what this `php.codecasts.rocks` source is? I am unfamiliar with it, and am not really able to find much info on it. Can't we use an official Alpine or PHP repository?